vfio-bindings/src/lib.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE-BSD-3-Clause file.
+
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+#[macro_use]
+extern crate vmm_sys_util;
+
+// generated with bindgen /usr/include/linux/vfio.h --constified-enum '*' --with-derive-default
+pub mod vfio;
+pub use crate::vfio::*;
+
+ioctl_io_nr!(VFIO_GET_API_VERSION, VFIO_TYPE, VFIO_BASE);
```

Comment:
Comparing this with the `kvm-ioctls` and `kvm-bindings` crates, this should maybe be part of a separate crate `vfio-ioctls`? Or part of the `vfio` crate itself, wdyt?

Reply:
I'm adding them to the vfio crate.